IPython/core/display.py

```diff
@@ -622,7 +645,7 @@ def _pngxy(data):
 def _jpegxy(data):
     """read the (width, height) from a JPEG header"""
     # adapted from http://www.64lines.com/jpeg-width-height
-    
+
```

Comment:
You can fix atom by going to: packages > whitespace > disable.

I'm mostly done objecting to PRs that do this, but I would always prefer that they not happen. I'm really disappointed in Atom's user-hostile aggressive defaults.